include/megaapi.h

```diff
@@ -12151,9 +12266,11 @@ class MegaApi
          * - MegaTextChatPeerList::PRIV_MODERATOR = 3
          * @param title Byte array representing the title that wants to be set, already encrypted and
          * converted to Base64url encoding (optional).
+         * @param unifiedKey Byte array that contains the unified key, already encrypted and
+         * converted to Base64url encoding.
          * @param listener MegaRequestListener to track this request
          */
-        void inviteToChat(MegaHandle chatid, MegaHandle uh, int privilege, const char *title = NULL, MegaRequestListener *listener = NULL);
+        void inviteToChat(MegaHandle chatid, MegaHandle uh, int privilege, const char *unifiedKey = NULL, const char *title = NULL, MegaRequestListener *listener = NULL);
```

Comment:
I'm not sure but, if `unifiedKey` is only related to public chats, it could be better to create a new function like `inviteToPublicChat` with this parameter mandatory and keeping `inviteToChat` in the previous state.